sambacc/config.py

```diff
@@ -56,6 +63,31 @@ class _FakeRefResolutionError(Exception):
     pass
 
 
+class ConfigFormatUnsupported(Exception):
+    pass
+
+
+if sys.version_info >= (3, 11):
+
+    def _load_toml(source: typing.IO) -> JSONData:
+        try:
+            import tomllib
+        except ImportError:
+            raise ConfigFormatUnsupported(ConfigFormat.TOML)
+        return tomllib.load(source)
+
+else:
+
+    def _load_toml(source: typing.IO) -> JSONData:
+        raise ConfigFormatUnsupported(ConfigFormat.TOML)
+
+
+def _detect_format(fname: str) -> ConfigFormat:
+    if fname.endswith(".toml"):
+        return ConfigFormat.TOML
+    return ConfigFormat.JSON
```

Comment:
OK, but I would prefer using  [splitext](https://docs.python.org/3/library/os.path.html#os.path.splitext) and compare with any of valid enum values of `ConfigFormat`. Just an idea.